setup.py

```diff
@@ -20,14 +20,15 @@ def run(self):
         super().run()
         if not self.dry_run:
             try:
-                _run(["tar", "xjf", "htslib.tar.bz2", "-C", "build"])
-                _run(["tar", "xjf", "bcftools.tar.bz2", "-C", "build"])
-                _run(["tar", "xjf", "samtools.tar.bz2", "-C", "build"])
-                _run(["./configure"], cwd="build/htslib")
-                _run(["./configure"], cwd="build/samtools")
+                os.makedirs('build/htslib', exist_ok=True)
+                _run(["tar", "-vxjf", "htslib.tar.bz2", "--directory=build/htslib", "--strip-components=1"])
+                os.makedirs('build/bcftools', exist_ok=True)
+                _run(["tar", "-vxjf", "bcftools.tar.bz2", "--directory=build/bcftools", "--strip-components=1"])
+                os.makedirs('build/samtools', exist_ok=True)
+                _run(["tar", "-vxjf", "samtools.tar.bz2", "--directory=build/samtools", "--strip-components=1"])
                 _run(["make"], cwd="build/htslib")
-                _run(["make"], cwd="build/bcftools")
                 _run(["make"], cwd="build/samtools")
+                _run(["make"], cwd="build/bcftools")
```

Comment:
This is needed to update the versions.

The `./configure` commands don't seem to be needed on the new versions, and were removed.

The internal folders have names like `htslib-1.15.1` instead of `htslib` and this prevents either 1. manually renaming the internal folder name and committing it each time we update or 2. renaming the directory once extracted.

All three libraries were updated at once, since they are all inter-related.